PRS2001-3000/PRS2907Brocchi.xml

```diff
@@ -44,14 +45,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                   Giovanni Battista
-                  Brocchi da Imola
-               </persName>
-               <occupation> dipl. </occupation>
+            <person sex="1" sameAs="wd:Q16639164">
+               <persName xml:lang="it" xml:id="n1">Giovanni Battista Brocchi da Imola</persName>
+               <persName xml:lang="la" xml:id="n2">Johannes Baptista Brocchus</persName>
+               <occupation type="political">Diplomat</occupation>
+               <floruit notBefore="1481" notAfter="1493">Nobleman and diplomat who accompanied an Ethiopian delegation from <placeName ref="LOC3957Jerusa">Jerusalem</placeName>
+                  to <placeName ref="LOC5374Rome">Rome</placeName> in <date>1481–1482</date>. Often conflated with his contemporary, the similarily named messenger
+                  <persName ref="PRS14337Baptista"/> (see the literature overview provided by Krebs).
+                  Contrary to that person, Giovanni Battista Brocchi da Imola most likely never visited Ethiopia.
+               </floruit>
             </person>
+            <listRelation>
+               <relation name="syriaca:different-from" active="PRS2907Brocchi" passive="PRS14337Baptista"/>
```

Comment:
on the contrary, if you insist on creating a second record, we need to show in the relation that they may rather be the same person.